### PR TITLE
add future release arg

### PR DESCRIPTION
**Pull request by rndmh3ro**

### CODE REVIEW

File: changelog.sh [hunk 1]
```diff
@@ -65,6 +65,11 @@ if [ ! -z "$INPUT_EXCLUDE_LABELS" ]; then
   ARG_EXCLUDE_LABELS="--exclude-labels $INPUT_EXCLUDE_LABELS"
 fi
 
+ARG_FUTURE_RELEASE=""
+if [ ! -z "$ARG_FUTURE_RELEASE" ]; then
```
Comment (charmixer) [inline changelog.sh]:
```suggestion
if [ ! -z "$INPUT_FUTURE_RELEASE" ]; then
```

File: changelog.sh [hunk 2]
```diff
@@ -65,6 +65,11 @@ if [ ! -z "$INPUT_EXCLUDE_LABELS" ]; then
...
+  ARG_FUTURE_RELEASE="--future-release ARG_FUTURE_RELEASE="
```
Comment (charmixer) [inline changelog.sh]:
```suggestion
  ARG_FUTURE_RELEASE="--future-release $INPUT_FUTURE_RELEASE"
```

File: changelog.sh [hunk 3]
```diff
@@ -65,6 +65,11 @@ if [ ! -z "$INPUT_EXCLUDE_LABELS" ]; then
...
+fi
+
 # build changelog
 echo "Running: github_changelog_generator $ARG_USER $ARG_PROJECT $ARG_RELEASE_BRANCH $ARG_BASE $ARG_OUTPUT $ARG_TOKEN $ARG_SINCE_TAG $ARG_DUE_TAG $ARG_EXCLUDE_TAGS $ARG_EXCLUDE_LABELS $ARG_DATE_FORMAT"
```
Comment (charmixer) [inline changelog.sh]:
```suggestion
echo "Running: github_changelog_generator $ARG_USER $ARG_PROJECT $ARG_RELEASE_BRANCH $ARG_BASE $ARG_OUTPUT $ARG_TOKEN $ARG_SINCE_TAG $ARG_DUE_TAG $ARG_EXCLUDE_TAGS $ARG_EXCLUDE_LABELS $ARG_DATE_FORMAT $ARG_FUTURE_RELEASE"
```

File: changelog.sh [hunk 4]
```diff
@@ -65,6 +65,11 @@ if [ ! -z "$INPUT_EXCLUDE_LABELS" ]; then
...
 github_changelog_generator $ARG_USER $ARG_PROJECT $ARG_RELEASE_BRANCH $ARG_BASE $ARG_OUTPUT $ARG_TOKEN $ARG_SINCE_TAG $ARG_DUE_TAG $ARG_EXCLUDE_TAGS $ARG_EXCLUDE_LABELS $ARG_DATE_FORMAT
```
Comment (charmixer) [inline changelog.sh]:
```suggestion
github_changelog_generator $ARG_USER $ARG_PROJECT $ARG_RELEASE_BRANCH $ARG_BASE $ARG_OUTPUT $ARG_TOKEN $ARG_SINCE_TAG $ARG_DUE_TAG $ARG_EXCLUDE_TAGS $ARG_EXCLUDE_LABELS $ARG_DATE_FORMAT $ARG_FUTURE_RELEASE
```

**Review by charmixer — APPROVED** [review]

**Comment by charmixer** [comment]
You can use `@ master` when using the github action instead of `@ v1` to use the new flag.

**Comment by rndmh3ro** [comment]
Thank you!

**Pull request merged**